lib/round.rb

```diff
@@ -0,0 +1,38 @@
+class Round
+  attr_reader :deck, :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @number_wrong = 0
```

Comment:
I'm curious why you decided to creat this number_wrong variable.. I don't see a need for it, but curious what your plan was for it.